third_party/NordicSemiconductor/drivers/radio/fem/nrf_fem_control_api.h

```diff
@@ -57,10 +57,10 @@ extern "C" {
 #define NRF_FEM_CONTROL_DEFAULT_LNA_PIN                     27
 
 /** Default PPI channel for pin setting. */
-#define NRF_FEM_CONTROL_DEFAULT_SET_PPI_CHANNEL             18
+#define NRF_FEM_CONTROL_DEFAULT_SET_PPI_CHANNEL             15
```

Comment:
Could you please align PPI channels also in [this file](https://github.com/openthread/openthread/blob/master/examples/platforms/nrf52840/platform-fem.h#L44).